### ISSUE TITLE
Service issue: SG mail server suddenly refusing emails

### ISSUE THREAD

**Issue by GCTempLogin** [eegeeZA/spamgourmet-clone#58]
Apologies in advance for bringing a service problem to the dev issue tracker. I posted on the BBS and sent a DM per the (very very old) sticky, but noticed that Josiah and SysKerr haven't been on it recently. Then I tried to mail info@ but that seems to be caught up in the same problem.  This was the closest thing to an alternative point of contact I could find (and forgive the throwaway login - Github is one of the services behind my SG aliases that isn't coming through, so I can't verify).

I noticed this morning that I could not reply to an SG masked reply address, nor could I send test messages to any of my aliases, existing or new.  The rejection was 

```
[<worktest.geekciro(at)0sg.net>](mailto:worktest.geekciro(at)0sg.net): host gourmet7.spamgourmet.com[216.75.62.102] said:
550-X-Host-Lookup-Failed: Reverse DNS lookup does not match for
<smarthost> 550 (deferred) (in reply to MAIL FROM command) 
```

The smarthost in question does have a PTR record/reverse dns set up, and has been working without issue for a couple of years.  I got the same rejection when I tried to mail to info@<spamgourmet> from my home or work accounts, and when I tried to email from gmail, removing my servers from the equation entirely, it got bounced with just "550 Incorrect Format" 

I don't know if there's something going on with gourmet7, or if something broke on my end?  

I know this is probably the wrong place, so even a pointer to a better contact point would be gratefully received.

**Comment by fredericmora** [comment]
I just tested that SG accepts emails from Gmail and forwards them to Gmail as well.

I suspect some problem related to that smarthost box. Can you give me a target test email address? I'll try to debug it.

**Comment by GCTempLogin** [comment]
Yeah, I thought gmail was working too, until I tried to mail info@  🤷‍♂️ 

worktest.geekciro(at)0sg.net   should be fine for testing. 

I dug into the smarthost and everything looks right. I was sure it was a me problem until I noticed a bunch of other mail not coming in, so at this point, I'm taking nothing for granted. :P

**Comment by GCTempLogin** [comment]
@fredericmora  I got the test email. Trying to reply to it gave me the same bounce.

**Comment by GCTempLogin** [comment]
@fredericmora  Confirm receipt of 2nd test message. Same result attempting to reply.

**Comment by fredericmora** [comment]
OK. This tells me that the problem is limited to the DNS check on emails incoming to SG, apparently only for the server you are using.

**Comment by GCTempLogin** [comment]
> OK. This tells me that the problem is limited to the DNS check on emails incoming to SG, apparently only for the server you are using.

By the server I'm using, do you mean my server, or gourmet7 or 0sg.net?

**Comment by fredericmora** [comment]
Yours. Whatever box is initiating the SMTP connection to gourmet7 when you try to send an email.

**Comment by GCTempLogin** [comment]
Ok... let me try something real quick... I'm going to change my protected address to proton and see if that works any better.

**Comment by GCTempLogin** [comment]
Changed that, taking my boxen out of the equation completely, and I'm still not getting my verify emails from github.

**Comment by fredericmora** [comment]
Let's move this discussion to the BBS. I replied to your thread.

**Comment by mh254** [comment]
Sorry to post here and not at the BBS, but i have the same issue - emails are rejected with DNS Lookup error and i did not get registration email from BBS.

**Comment by fredericmora** [comment]
mh254: Could you please try again? We updated the nameserver and the lookup errors should be gone.

**Comment by lwcorp** [comment]
Still doesn't work for me, as sender from 5 minutes ago gets:
> Your message wasn't delivered because the address couldn't be found, or is unable to receive mail.
> 550 X-Host-Lookup-Failed: Reverse DNS lookup does not match for 209.85.167.49 (deferred)

```
C:\>ping -n 1 -a 209.85.167.49
Pinging mail-lf1-f49.google.com [209.85.167.49]
```

**Comment by lwcorp** [comment]
Looks like messages are back! But what about all the old messages? Are they gone for good?

**Comment by fredericmora** [comment]
It depends where the old messages were held. Can you give us an update if you have old messages trickling?

**Comment by lwcorp** [comment]
Nothing, as if they fell into a black hole...quite disturbing really.

**Comment by manglewit** [comment]
I don't know if this is a continuation of the problem, or something different, but I'm still not getting messages via spamgourmet. The 'remaining' counter is decreasing on the spamgourmet addresses under the 'Advanced' tab, but nothing coming through to the protected address mailbox, which is on runbox.com ( https://runbox.com/ ). I'm reluctant to raise a ticket with runbox until I know that things are working properly with spamgourmet, as messages stopped around on or after 16th Jan, which seems to coincide with the (?solved?) DNS issue. Is spamgourmet running nominally?

**Comment by lwcorp** [comment]
I suggest change to Gmail at least temporarily because over the forums many claim it works there.
If it does work for you, obviously runbox checks something Gmail doesn't care about.
It's far from ideal but at least you'll stop losing messages until it's really handled.

**Comment by fredericmora** [comment]
Can you PM josh or syskoll on the BBS and send the private runbox.com
address that cannot get emails from sg?
